Makefile

```diff
@@ -13,13 +13,13 @@ stop:	## Stop the environment
 	docker-compose stop
 
 bash:	## Go to the backend container
-	docker exec -ti codeforpoznanpl_v3_backend_1 bash
+	docker exec -ti cfp_v3_backend_1 bash
```

Comment:
Here we can use `docker-compose exec` with the namees `frontend`, `backend`, etc. Just please test it. ;-)